ltx_video/pipelines/pipeline_ltx_video.py

```diff
@@ -370,10 +372,10 @@ def encode_prompt(
                 return_tensors="pt",
             )
             negative_prompt_attention_mask = uncond_input.attention_mask
-            negative_prompt_attention_mask = negative_prompt_attention_mask.to(device)
+            negative_prompt_attention_mask = negative_prompt_attention_mask.to(text_enc_device)
 
             negative_prompt_embeds = self.text_encoder(
-                uncond_input.input_ids.to(device),
+                uncond_input.input_ids.to(text_enc_device),
                 attention_mask=negative_prompt_attention_mask,
             )
             negative_prompt_embeds = negative_prompt_embeds[0]
```

Comment:
why is the device at line 388 still using "device" and not "text_enc_device", is that on purpose? i'm trying to implement this on my local code and I saw that

Reply:
i only used text_enc_device to move the tensors that are used as input for the text_encoder to the device that the text encoder is on (just to make sure they are on the same). 

later in the code in line 388 it is not used anymore since it has already been through the text_encoder. (and also bc it didn't cause any errors during my tests lol)